src/audio/copier/copier_dai.c

```diff
@@ -99,8 +99,6 @@ static int copier_dai_init(struct comp_dev *dev,
 		pipeline->source_comp = dev;
 	}
 
-	dev->ipc_config.frame_fmt = config->frame_fmt;
```

Comment:
this needs to be a separate patch with an explanation for why it is not needed anymore

Reply:
done